data/presets/presets/leisure/fishing.json

```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "maki-aquarium",
```

Comment:
I am not certain this is the best icon here, perhaps a new icon needs to be created.

Reply:
I agree - you can use [`fas-fish`](https://fontawesome.com/icons/fish?style=solid) for this..